virttest/utils_test/__init__.py

```diff
@@ -1755,46 +1755,44 @@ def __str__(self):
 class VMStress(object):
 
     """
-    Run Stress tool in vms, such as stress, unixbench, iozone and etc.
+    Run Stress tool in vms, such as stress, iozone and etc.
     """
 
-    def __init__(self, vm, stress_type):
-        """
-        Set parameters for stress type
+    def __init__(self, vm, stress_type, download_url="", make_cmds="",
```

Comment:
`utils_test.load_stress(stress_type, vms, params):` should be made aware of these function params for tests to make use of it incase need.